dm/syncer/error.go

```diff
@@ -54,6 +54,19 @@ func ignoreDDLError(err error) bool {
 	}
 }
 
+// ignoreTrackerDDLError is also same with ignoreDDLError, but in order to keep tracker's table structure same as
+// upstream's, we can ignore "already exists" errors because already exists doesn't mean same.
+func ignoreTrackerDDLError(err error) bool {
+	switch {
+	case infoschema.ErrDatabaseExists.Equal(err), infoschema.ErrDatabaseDropExists.Equal(err),
```

Comment:
can you give a example to explain why we can ignore the `ErrDatabaseExists/ErrDatabaseDropExists/ErrTableExists` error?

i mean i can't understand this `we can ignore "already exists" errors because already exists doesn't mean same.`

does that mean when i want to sync 'create/drop database/table a' to downstream, we can countinue to sync regardless of whether the database/table existes or not?

Reply:
sorry i want to say “we can’t ignore“😂